src/components/eventCard.js

```diff
@@ -69,8 +69,8 @@ function Content({type, width, size, event, power}) {
         }
     }, [event, transferCount])
 
-    const nl2br = (text) => (text.split('\n').map((item, key) => {
-        return <>{item}<br/></>
+    const nl2br = (text) => (text.split('\n').map(item => {
+        return <React.Fragment key={item.toString()}>{item}<br/></React.Fragment>
```

Comment:
here we can use tags `<>`  and `</>` to replace React.Fragment and simplify the code

Reply:
to be able to add key prop

Reply:
to be able to add key prop

Reply:
ok, this must be a snippet of something in particular? Could it be handled with a <p> tag to make it semantically more correct in html code?